src/lib/filter-style-values.ts

```diff
@@ -3,38 +3,35 @@
  *
  * - IE 5, 6, and 7 support computed properties via injecting JS with expression().
  * - JS can be executed in place of a url():
- *   -  backround-color: "javascript:alert(1)"
+ * - background-color: "javascript:alert(1)"
```

Comment:
I don't think this was supposed to be modified.

Reply:
oh good catch, my b